src/pages/instances/InstanceOverviewDevices.tsx

```diff
@@ -52,6 +64,53 @@ const InstanceOverviewDevices: FC<Props> = ({ instance }) => {
           role: "rowheader",
           "aria-label": "Type",
         },
+        {
+          content: (
+            <div>
+              {device.type == "disk" ? (
+                isRootDisk(device as FormDevice) ? (
+                  <>
+                    <strong>Pool: </strong>
+                    <>{device.pool}</>
+                  </>
+                ) : (
+                  <>
+                    <strong>Storage Volume:</strong>
+                    <Link
+                      to={`/ui/project/${instance.project}/storage/pool/${device.pool}/volumes/custom/${device.source}`}
+                    >
+                      {` ${device.source}`}
+                    </Link>
+                  </>
+                )
+              ) : (
+                ""
+              )}
+              {device.type === "gpu" && (
+                <>
+                  <strong>Pci: </strong>
+                  <>{device.pci}</>
+                </>
+              )}
+              {device.type === "proxy" &&
+                deviceDetailRows([
```

Comment:
I think it's simpler to just define two `div` elements here instead doing a function that internally is doing a map render.